draft-ietf-oauth-status-list.md

```diff
@@ -136,38 +136,38 @@ The following diagram depicts the relationship between the involved roles (Relyi
 
 This specification seeks to find a balance between scalability, security, and privacy by minimizing the status information to mere bits (often a single bit) and compressing the resulting binary data. Thereby, a Status List may contain statuses of many thousands or millions Referenced Tokens while remaining as small as possible. Placing large amounts of Referenced Tokens into the same list also enables herd privacy relative to the Status Provider.
 
 ## Design Considerations
 
 The decisions taken in this specification aim to achieve the following design goals:
 
-* the specification shall favor a simple and easy to understand concept
-* the specification shall be easy, fast and secure to implement in all major programming languages
+* the specification shall favor a simple and easy-to-understand concept
+* the specification shall be easy, fast, and secure to implement in all major programming languages
 * the specification shall be optimized to support the most common use cases and avoid unnecessary complexity of corner cases
-* the Status List shall scale up to millions of tokens to support large scale government or enterprise use cases
+* the Status List shall scale up to millions of tokens to support large-scale government or enterprise use cases
 * the Status List shall enable caching policies and offline support
 * the specification shall support JSON and CBOR based tokens
 * the specification shall not specify key resolution or trust frameworks
 * the specification shall design an extension point to convey information about the status of a token that can be re-used by other mechanisms
 
 ## Status Mechanism Registry
 
-This specification establishes the IANA "Status Mechanisms" registry for status mechanism and registers the members defined by this specification. Other specifications can register other members used for status retrieval. Other status mechanisms may have different tradeoffs regarding security, privacy, scalability adn complexity. The privacy and security considerations in this document only represent the properties of the Status List mechanism.
+This specification establishes the IANA "Status Mechanisms" registry for status mechanisms and registers the members defined by this specification. Other specifications can register other members used for status retrieval. Other status mechanisms may have different tradeoffs regarding security, privacy, scalability, and complexity. The privacy and security considerations in this document only represent the properties of the Status List mechanism.
```

Comment:
Same here we either need to use oxford comma's consistently or not, I would suggest not.